Assets/MixedRealityToolkit/Utilities/Physics/TwoHandScaleLogic.cs

```diff
@@ -25,34 +25,32 @@ public class TwoHandScaleLogic
         /// <summary>
         /// Initialize system with source info from controllers/hands
         /// </summary>
-        /// <param name="handsPressedMap">Dictionary that maps inputSources to states</param>
+        /// <param name="handsPressedArray">Dictionary that maps inputSources to states</param>
         /// <param name="manipulationRoot">Transform of gameObject to be manipulated</param>
-        public virtual void Setup(Dictionary<uint, Vector3> handsPressedMap, Transform manipulationRoot)
+        public virtual void Setup(Vector3[] handsPressedArray, Transform manipulationRoot)
         {
-            startHandDistanceMeters = GetMinDistanceBetweenHands(handsPressedMap);
+            startHandDistanceMeters = GetMinDistanceBetweenHands(handsPressedArray);
             startObjectScale = manipulationRoot.transform.localScale;
         }
 
         /// <summary>
         /// update GameObject with new Scale state
         /// </summary>
         /// <returns>a Vector3 describing the new Scale of the object being manipulated</returns>
-        public virtual Vector3 UpdateMap(Dictionary<uint, Vector3> handsPressedMap)
+        public virtual Vector3 UpdateMap(Vector3[] handsPressedArray)
         {
-            var ratioMultiplier = GetMinDistanceBetweenHands(handsPressedMap) / startHandDistanceMeters;
+            var ratioMultiplier = GetMinDistanceBetweenHands(handsPressedArray) / startHandDistanceMeters;
             return startObjectScale * ratioMultiplier;
         }
 
-        private float GetMinDistanceBetweenHands(Dictionary<uint, Vector3> handsPressedMap)
+        private float GetMinDistanceBetweenHands(Vector3[] handsPressedArray)
```

Comment:
>GetMinDistanceBetweenHands [](start = 22, length = 26)

Not that you should actually do this:

[https://en.wikipedia.org/wiki/Closest_pair_of_points_problem](https://en.wikipedia.org/wiki/Closest_pair_of_points_problem)

Reply:
I did think this, but it may be a bit excessive when generally we're talking about no more than two points, haha.